lib/Sema/CSApply.cpp

```diff
@@ -1615,6 +1620,66 @@ namespace {
         return forceUnwrapIfExpected(ref, memberLocator);
       }
 
+      if (requiresDistributedThunk(base, memberLoc.getStartLoc(), memberRef)) {
+        auto *decl = memberRef.getDecl();
+        FuncDecl *thunkDecl = nullptr;
+        if (auto *FD = dyn_cast<FuncDecl>(decl)) {
+          thunkDecl = FD->getDistributedThunk();
+        } else {
+          thunkDecl = cast<VarDecl>(decl)->getDistributedThunk();
```

Comment:
maybe 
```suggestion
          } else if (let var cast<VarDecl>(decl)) { 
            thunkDecl = var->getDistributedThunk();
          } else llvm_unreachable...
```

? Just in case I guess...

Reply:
The validity is checked by `requiresDistributedThunk` so I don't really see a reason to double-check here, if anything cast is going to fail.

Reply:
Gotcha, thanks!